R/col-data.R

```diff
@@ -27,6 +28,11 @@ print.cf_data <- function(x, ...) {
   print(format(x, ...))
 }
 
+#' @export
+#' @param width The maximum column width
+#' @param align Alignment of the column
+#' @param min_width The minimum allowed column width, `width` if omitted
+#' @rdname cf_data
```

Comment:
This should have an argument `subclass` so it's clear that this is what you use when create subclasses

Reply:
And it might be worthwhile to think through the docs with an eye to making it more clear that this is what you should implement if you want to make it work for your own class.

Reply:
Add `...` to `new_cf_data()` .